rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cat.allocation/10_basic.yml

```diff
@@ -194,7 +194,7 @@
   - match:
       $body: |
             /^
-              ( 0                   \s+
+              ( \d                  \s+    #usually 0, unless some system index has been recreated before this runs
```

Comment:
What's strange to me is that we do this above in the "all nodes" test with a `\d+`, but then in the "column headers" test we also assert that it's 0, so I think we'd need to change that one also if we went with this fix.

Reply:
Good point. I added that. What's going to be trickier (and maybe make us rethink these tests) are failures like https://github.com/elastic/elasticsearch/issues/84765. That's doing a call to `_cat/indices`, and failing because it's getting back an unexpected second row for the index `.tasks`.